packages/coil-extension/package.json

```diff
@@ -67,7 +67,7 @@
     "@webmonetization/polyfill-utils": "^0.0.0",
     "@webmonetization/types": "^0.0.0",
     "@webmonetization/wext": "^0.0.0",
-    "debug": "^4.3.1",
+    "debug": "patch:debug@npm%3A4.3.4#~/.yarn/patches/debug-npm-4.3.4-4513954577.patch",
```

Comment:
just leave this is as normal, you only need to touch the root package.json

the `resolutions` field does an automagical override